src/DeviceInterface/InterfacePrototype.js

```diff
@@ -778,9 +778,8 @@ class InterfacePrototype {
     preAttachTooltip (topContextData, input, form) {
         // A list of checks to determine if we need to generate a password
         const checks = [
-            topContextData.inputType === 'credentials.password',
-            this.settings.featureToggles.password_generation,
-            form.isSignup
+            topContextData.inputType === 'credentials.password.new',
```

Comment:
No need to rely on `form.isSignup` because we read the the `password.new` attribute instead.